src/.vuepress/config.js

```diff
@@ -440,6 +440,11 @@ module.exports = {
             "Build gaze-contingent assistive applications with Neon",
           path: "gaze-contingency-assistive",
         },
+        {
```

Comment:
@tpfffr, I added this to the config.js so that the article shows on the side/titlebar. Feel free to edit the title.